modules/debugger.py

```diff
@@ -453,6 +453,12 @@ async def resume(self) -> None:
 		try: await self.active.resume()
 		except core.Error as e: self.console.error(f'Unable to continue: {e}')
 
+	@core.run
+	async def reverse_continue(self) -> None:
+		try: await self.active.reverse_continue()
+		except core.Error as e:
+			self.console.error(f'Unable to reverse continue: Try running the "Debugger: Start recording for reverse debugging" command first. Details: {e}')
```

Comment:
This seems like a GDB specific thing I don't think suggesting fixes is something we should be doing the adapter should be issuing a relevant error message

Reply:
done, but I found the DAP message a bit confusing:

`Could not step back: Target multi-thread does not support this command. (from exec-step --reverse) - Try running 'target record-full' before stepping back`